src/_nms_api_releases/nms-api-01-08-01-release.md

```diff
@@ -0,0 +1,8 @@
+---
+version: 1.8.1
+release_date: 2022-05-29
+release_notes: https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12349546&styleName=&projectId=12311201&Create=Create&atl_token=A5KQ-2QAV-T4JA-FDED_ed4974105fc0907a610ea3bcd3a0d26fc996a2d4_lout
```

Comment:
The release notes link seems overly complicated, projectId and version should be all that's needed.